lib/internal/Magento/Framework/Interception/PluginList/PluginList.php

```diff
@@ -205,14 +206,12 @@ protected function _inheritPlugins($type)
     /**
      * Trims starting backslash from plugin instance name
      *
-     * @param array $plugins
+     * @param array $plugin
      * @return void
      */
-    private function trimInstanceStartingBackslash(&$plugins)
+    private function trimInstanceStartingBackslash(&$plugin)
```

Comment:
I'm fine with that. I only kept it this way to try to avoid changing more than I needed. Do you think  it's worth this being a separate method at all? It's private and not used otherwise, so it might be a good candidate for inlining. https://refactoring.guru/inline-method

Reply:
Feel free to remove this method. I do not think, in this case, a single line method helps anything here.